src/test/shell/bazel/apple/bazel_objc_test.sh

```diff
@@ -117,4 +117,60 @@ EOF
       fail "Timestamp of contents of archive file should be zero"
 }
 
+function test_strip_symbols() {
+  setup_objc_test_support
+
+  rm -rf ios
+  mkdir -p ios
+
+  cat > ios/unused.h <<EOF
+#import <Foundation/Foundation.h>
+@interface UnusedClass: NSObject
+@property (nonatomic, assign) bool someProperty;
+@end
+EOF
+
+  cat > ios/unused.m <<EOF
+#import "unused.h"
+@implementation UnusedClass
+@end
+EOF
+
+  cat >ios/main.m <<EOF
+#import <UIKit/UIKit.h>
+#import "unused.h"
+
+int main(int argc, char *argv[]) {
+  NSAutoreleasePool * pool = [[NSAutoreleasePool alloc] init];
+  int retVal = UIApplicationMain(argc, argv, nil, nil);
+  [pool release];
+  return retVal;
+}
+EOF
+
+  cat >ios/BUILD <<EOF
+apple_binary(name = 'app',
+             deps = [':main'],
+             platform_type = 'ios')
+
+objc_library(name = 'main',
+             non_arc_srcs = ['main.m'],
+             deps = [':lib'])
+
+objc_library(name = 'lib',
+             srcs = ['unused.m'],
+             hdrs = ['unused.h'])
+
+EOF
+
+  bazel build -s --verbose_explanations --verbose_failures \
+      --ios_sdk_version=$IOS_SDK_VERSION \
+      --objc_enable_binary_stripping=true \
+      --compilation_mode=opt \
+      //ios:app >$TEST_log 2>&1 || fail "should pass"
+
+  ls bazel-out/apl-ios_x86_64-opt/bin/ios/app_lipobin \
```

Comment:
Where does the output get the `_lipobin` suffix from?

Reply:
I believe that is the formatted name for a single architecture binary